Django-CoWork-Connect/coapp/forms.py

```diff
@@ -5,34 +5,8 @@
 class ReservationForm(forms.ModelForm):
     class Meta:
         model = Reservation
-        fields = ['desk', 'start_date', 'end_date']
-        labels = {
-            'desk': 'Dostępne biurka',
-            'start_date': 'Data początkowa',
-            'end_date': 'Data końcowa',
-        }
+        fields = ['start_date', 'end_date']
         widgets = {
-            'start_date': forms.DateInput(attrs={'class': 'datepicker', 'type': 'text', 'autocomplete': 'off'}),
-            'end_date': forms.DateInput(attrs={'class': 'datepicker', 'type': 'text', 'autocomplete': 'off'}),
+            'start_date': forms.DateInput(attrs={'type': 'date'}),
+            'end_date': forms.DateInput(attrs={'type': 'date'}),
         }
-
-    def __init__(self, *args, **kwargs):
```

Comment:
czemu to usuwamy ?

Reply:
dlatego, że na poczatku był osobny formularz do rezerwacji w menu, a teraz to wszystko jest w opcji dostepności biurek i teraz jest to zbędne